test/cpp/microbenchmarks/bm_fullstack.cc

```diff
@@ -642,6 +830,27 @@ BENCHMARK_TEMPLATE(BM_PumpStreamServerToClient, SockPair)
 BENCHMARK_TEMPLATE(BM_PumpStreamServerToClient, InProcessCHTTP2)
     ->Range(0, 128 * 1024 * 1024);
 
+// Generate Args for StreamingPingPong benchmarks. Currently generates args for
+// only "small streams" (i.e streams with 0, 1 or 2 messages)
+static void StreamingPingPongArgs(benchmark::internal::Benchmark* b) {
+  int msg_size = 0;
+  int num_ping_pongs = 0;
+  for (msg_size = 0; msg_size <= 128 * 1024 * 1024;
+       msg_size == 0 ? msg_size++ : msg_size *= 8) {
+    for (num_ping_pongs = 0; num_ping_pongs <= 2; num_ping_pongs++) {
```

Comment:
Suggest starting num_ping_pongs at 1 if msg_size != 0: the num_ping_pongs==0 benchmark case is the same regardless of message size (since it sends no message)